src/cargo/ops/cargo_rustc/job_queue.rs

```diff
@@ -209,12 +212,18 @@ impl<'a> JobQueue<'a> {
             // Any dirty stage which runs at least one command gets printed as
             // being a compiled package
             Dirty => {
-                self.printed.insert(pkg);
-                try!(config.shell().status("Compiling", pkg));
+                //check if documenting or compiling
```

Comment:
Ah it's ok to leave out a comment like this, it pretty clearly follows from the code below!